src/github.py

```diff
@@ -1,87 +1,96 @@
 import os
 import re
 from collections import defaultdict, namedtuple
-from typing import Any, Generator
-
+from typing import Any, Sequence, Generator
+import dataclasses
 import httpx
 from markdown_it import MarkdownIt
 from markdown_it.tree import SyntaxTreeNode
 
-Issue = namedtuple("Issue", "metadata body")
 
+def get_content_issues(
+    body: dict[str, Any], 
+    issues_tag: str,
+)  -> list[int]:
+    """
+    Returns Issues Passed in sections of the issue body
+    """
+    issues = getattr(body, issues_tag, "")
+    return [int(n) for n in re.findall(r"\d+", issues)]
 
-class Episode:
-    def __init__(self, episode_id: int, issue_fields: list):
-        self.episode_id = episode_id
-        self.raw = get_from_github(self.episode_id)
-        self.body = parse_issue_markdown(self.raw["body"])
 
-        for key, value in self.body.items():
-            setattr(self, key, value)
+def parse_issue_markdown(text) -> dict[str, str]:
+    """Use markdownit to split at section headings"""
+    md = MarkdownIt("zero", {"maxNesting": 1})
+    md.enable(["heading", "paragraph"])
+    tokens = md.parse(text)
+    node = SyntaxTreeNode(tokens)
+    issue_object = defaultdict(list)
+    for n in node.children:
+        if n.type == "heading":
+            issue_key = n.children[0].content.lower().replace(" ", "_")
+        elif content := n.children[0].content == "_No response_":
+            continue
+        else:
+            issue_object[issue_key].append(n.children[0].content)
 
-        for field in issue_fields:
-            setattr(self, field, list(get_issues(get_content_issues(self.body, field))))
+    return {key: "\n".join(value) for key, value in issue_object.items()}
+        
 
-    @property
-    def title(self) -> str:
-        return self.raw["title"]
+@dataclasses.dataclass
```

Comment:
The addition of this dataclass was due to the multiple repo layout. Now you're able to pull topics by providing the owner/repo.